lasagne/layers/prelu.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import theano.tensor as T
+
+from .. import init
+from .. import nonlinearities
+
+from .base import Layer
+
+__all__ = [
+    "PReLULayer"
+]
+
+class PReLULayer(Layer):
+    """
+    A layer used to implement Parametric Rectified Linear Units (PReLU).
+    This layer simply applies the parametric nonlinearity to the previous
+    layer's output. 
+
+    Parameters
+    -----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape.
+
+    alpha : double or None
+        The initial "leakiness" of the rectifier which is then learned.
+        If argument not provided, initial leakiness will be set to 0.25.
+
+    unique_chan_param : boolean or None
```

Comment:
Why should this be used only in the case of a convolutional layer? I see no objections to using this in combination with a dense layer (where each unit is just considered a separate channel). It will probably overfit completely but technically there's nothing wrong with it :) Or am I missing something? Also I'm not too sure about `unique_chan_param` as a name. I made a throwaway implementation of this a while back and I think I called it `tied`, which may be a little too concise :) ('tied' as in the parameters are tied across all channels, or not). Anybody have a better idea?